src/main/java/MeetingOrganizer.java

```diff
@@ -0,0 +1,172 @@
+import com.google.gson.JsonArray;
```

Comment:
![image](https://user-images.githubusercontent.com/59819947/78003678-a4b14f80-736b-11ea-9c7b-cbabae25a62e.png)
Should there be return lines added?